dashboard/models.py

```diff
@@ -31,12 +31,12 @@ class AcademicTerms(models.Model):
     date_start = models.DateTimeField(verbose_name="Start Date and Time", blank=True, null=True)
     date_end = models.DateTimeField(verbose_name="End Date and Time", blank=True, null=True)
 
-    def __str__(self):
+    def __str__(self) -> models.CharField:
```

Comment:
`self.name` returns `<class 'str'>`.

Reply:
Yeah I remember I was supposed to look at the return types for these models.